Makefile

```diff
@@ -24,15 +24,18 @@ endif
 
 # Targets to make
 
-images: package/.image.lighthouse-agent
-	./scripts/build-coredns $(coredns) $(BUILD_ARGS)
+images: package/.image.lighthouse-agent package/.image.lighthouse-coredns
 
 # Explicitly depend on the binary, since if it doesn't exist Shipyard won't find it
 package/.image.lighthouse-agent: bin/lighthouse-agent
 
+package/.image.lighthouse-coredns: bin/lighthouse-coredns
+
 bin/lighthouse-agent: vendor/modules.txt $(shell find pkg/agent)
 	${SCRIPTS_DIR}/compile.sh $@ pkg/agent/main.go
 
+bin/lighthouse-coredns: vendor/modules.txt $(shell find pkg/coredns)
+	${SCRIPTS_DIR}/compile.sh $@ pkg/coredns/main.go
 deploy: images clusters
```

Comment:
Please add a blank line here ;-).

Reply:
Thanks @skitt. yes, debian was the image we were using so far. I will open another PR for ubi as it needs its own tests. what do you think?